rakelib/docs.rake

```diff
@@ -200,6 +204,14 @@ begin
       inventory   = Bolt::Config::INVENTORY_OPTIONS.dup
       @transports = Bolt::Config::TRANSPORT_CONFIG.keys
 
+      # Move sub-options for 'log' option up one level, as they're nested under
+      # 'console' and filepath
+      @opts['log'][:properties] = @opts['log'][:additionalProperties][:properties]
+
+      # Remove 'notice' log level. This is soft-deprecated and shouldn't appear
+      # in documentation.
```

Comment:
While we're here, can we update https://github.com/puppetlabs/bolt/blob/main/documentation/templates/bolt_defaults_reference.md.erb#L9 to have 1 space instead of 2?